src/clj/rems/api/permissions.clj

```diff
@@ -14,11 +15,13 @@
     :tags ["permissions"]
 
     (GET "/:user" []
-      :summary "Returns user's permissions in ga4gh visa format. See https://github.com/ga4gh-duri/ga4gh-duri.github.io/"
+      :summary "Experimental. Returns user's permissions in ga4gh visa format. Currently signed with fake key. See https://github.com/ga4gh-duri/ga4gh-duri.github.io/"
       :roles #{:handler :owner}
       :path-params [user :- (describe s/Str "return permissions for this user, required")]
       :query-params [{expired :- (describe s/Bool "whether to include expired permissions") false}]
       :return GetPermissionsResponse
-      (when-not (empty? user)
-        (api-util/not-found-json-response))
-      (ok (entitlements/get-entitlements-for-permissions-api user nil expired)))))
+      (if (not (:enable-permissions-api config/env))
+        (not-implemented "permissions api not implemented")
+        (do (when-not (empty? user)
+              (api-util/not-found-json-response))
```

Comment:
Heh, I just noticed that this `when` does nothing. `(api-util/not-found-json-response)` is a value which just gets thrown away. You probably meant:

```
(if-not (empty? user)
  (api-util/not-found-json-response)
  (ok (entitlements/get-entitlements-for-permissions-api user nil expired)))
```

but even that has the condition flipped around, so probably

```
(if (empty? user)
  (api-util/not-found-json-response)
  (ok (entitlements/get-entitlements-for-permissions-api user nil expired)))
```

On the other hand, since this `when-not` is not doing anything and the tests pass, you can probably just remove it :sunglasses: